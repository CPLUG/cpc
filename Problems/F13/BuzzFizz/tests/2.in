10
2
1 Javascipt
2 Sucks
