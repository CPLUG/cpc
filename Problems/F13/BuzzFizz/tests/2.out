Javascipt
JavasciptSucks
Javascipt
JavasciptSucks
Javascipt
JavasciptSucks
Javascipt
JavasciptSucks
Javascipt
JavasciptSucks
